### ISSUE TITLE
iOS has an entirely different check mark...

### ISSUE THREAD

**Issue by phet-steele** [phetsims/unit-rates#180]
...with an entirely different drop shadow to boot. @arouinfar is that okay? The same thing occurs in Balancing Chemical Equations and no one seems to care, but I'm going to mention it anyway.

Expected:
<img width="512" alt="screen shot 2017-03-08 at 12 07 20 pm" src="https://cloud.githubusercontent.com/assets/5863899/23719336/32caa3d4-03f8-11e7-96e6-e6f9a19539d1.png">

iOS:
![img_0073](https://cloud.githubusercontent.com/assets/5863899/23719294/0de71854-03f8-11e7-960a-b32faec86a4d.PNG)

For phetsims/tasks/issues/797.

**Comment by arouinfar** [comment]
@phet-steele good observation. I find it a little odd, but not bothersome. 

Perhaps there could be a separate common code issue to look into the difference in appearance, and perhaps address it? I don't think it's a high priority issue, but I still think it would be good to look into it, as time allows. 

@phet-steele can you create an issue about the checkmark appearance in the appropriate common code repo?

**Comment by pixelzoom** [comment]
Curious...

The checkmark is a Unicode character (`'\u2713'`, "heavy-ballot check mark") that is rendered with `scenery-scenery.ShadowText`, with font family `'Arial'`.    Here's the code in ShoppingQuestionNode:

```js
// check mark to right of box, to indicate that the question has been correctly answered
var checkMarkNode = new ShadowText( '\u2713', {
  fill: URColors.checkMark,
  font: new URFont( 36 ),
  ...
} );
```

But it appear to be rendered in a different font on iOS. 

@jonathanolson Any insights?

**Comment by jonathanolson** [comment]
> The checkmark is a Unicode character ('\u2713') that is rendered with scenery-scenery.ShadowText, with font family 'Arial'.

Doesn't look like the Mac version of Arial on my MBP specifies the question mark, probably falls back to the  default "sans-serif" system font per platform.

If a consistent appearance is desired, font-awesome could be used?

**Comment by pixelzoom** [comment]
@arouinfar Is the font-awesome check marker acceptable? http://fontawesome.io/icon/check/

**Comment by pixelzoom** [comment]
In context (I can investigate whether there's a version without the box):

![screenshot_171](https://cloud.githubusercontent.com/assets/3046552/23725367/7bdd3090-040d-11e7-9839-7790577b81f7.png)

**Comment by pixelzoom** [comment]
Here's the font-awesome check mark without the box.  I don't think it would benefit from a drop shadow. @arouinfar is this acceptable?

![screenshot_175](https://cloud.githubusercontent.com/assets/3046552/23725785/2614496c-040f-11e7-9b4b-796c703a9e88.png)

**Comment by pixelzoom** [comment]
Other sims that use the Unicode check mark ('\u2713'), and should also be fixed:

balancing-chemical-equations
graphing-lines

**Comment by arouinfar** [comment]
@pixelzoom I like the checkmark in https://github.com/phetsims/unit-rates/issues/180#issuecomment-285182113. As for the drop shadow, I could take it or leave it. 

Since this will effect more than Unit Rates, I'd like to for @ariel-phet and/or @amanda-phet to also review.

**Comment by amanda-phet** [comment]
I think the other sims also have an incorrect "x", so if we made a universal change we'd want to consider how that would look we well. 

The default sans-serif in iOS doesn't bother me, but using fontawesome seems like a good idea. 

This is the "x" that we'd probably use:
![screen shot 2017-03-08 at 3 13 29 pm](https://cloud.githubusercontent.com/assets/7852916/23726557/deebf4d8-0411-11e7-99b0-81a8a1059f7b.png)

**Comment by phet-steele** [comment]
> I think the other sims also have an incorrect "x", so if we made a universal change we'd want to consider how that would look we well.

Believe it or not, the "x" is the same on both platforms. The "x" in question is below and looks the same on iOS vs. macOS 10.12.3, maybe something else is going on here?
<img width="378" alt="screen shot 2017-03-08 at 3 21 04 pm" src="https://cloud.githubusercontent.com/assets/5863899/23726843/0b361cde-0413-11e7-9b9b-af0dccd9eec5.png">

Regardless, @amanda-phet is correct in saying that if we change the check mark, that means we should change the "x" as well.

**Comment by pixelzoom** [comment]
Indeed, those other sims use Unicode "big X" (`'\u2718'`) for incorrect.  E.g.:

![screenshot_177](https://cloud.githubusercontent.com/assets/3046552/23726992/93ebc81c-0413-11e7-9e75-00caad99565f.png)

**Comment by pixelzoom** [comment]
Fontawesome check marks are used in this dev version, if anyone wants to test drive:
http://www.colorado.edu/physics/phet/dev/html/unit-rates/1.0.0-dev.69/unit-rates_en.html

**Comment by amanda-phet** [comment]
The check looks fine to me, and as long as people are OK with the x then we can plan to use those icons in future sims (and sim releases).

**Comment by ariel-phet** [comment]
I would say the non-font-awesome has a bit more of a "hand drawn feel" so it is really an aesthetic choice. I am fine with either, and I think the meaning is clear with either. 

Deferring that decision to @arouinfar since @amanda-phet seems fine with the look

**Comment by pixelzoom** [comment]
@arouinfar If you're fine with the font-awesome icons (check and X), then assign back to me. I'll create issues to use those icons in balancing-chemical-equations and graphing-lines.

**Comment by arouinfar** [comment]
@ariel-phet I would agree that the original (non-iOS) check is a bit more aesthetically pleasing than the font-awesome check. However, I think the advantage of using the font-awesome icons will be that the look is consistent across platforms, which is apparently a failing of the unicode version.

@pixelzoom please proceed with the font-awesome icons.

**Comment by pixelzoom** [comment]
Implemented for unit-rates in [1.0.0-dev.69](http://www.colorado.edu/physics/phet/dev/html/unit-rates/1.0.0-dev.69/unit-rates_en.html).  Issues created for other sims. Closing.
